tools/topology/topology2/cavs-mixin-mixout-hda.conf

```diff
@@ -36,11 +36,34 @@ Object.Pipeline {
 				stream_name $HDA_ANALOG_DAI_NAME
 				dai_type "HDA"
 				copier_type "HDA"
+
+				num_mod_cfgs 1
+				Object.Base.mod_cfg.1 {
+					ibs 123
+					obs 456
+					cpc 789
+				}
 			}
 			Object.Widget.gain.1 {
 				Object.Control.mixer.1 {
 					name '2 Main Playback Volume'
 				}
+				Object.Base.mod_cfg.1 {
+					ibs 1123
+					obs 1456
+					cpc 1789
+				}
+				Object.Base.mod_cfg.2 {
+					ibs 2112
+					obs 2145
+					cpc 2178
+				}
+				Object.Base.mod_cfg.3 {
+					ibs 3112
+					obs 3145
+					cpc 3178
+				}
+				num_mod_cfgs 3
```

Comment:
Do we need this, the CC should count this.

Reply:
Ok. I was actually wondering why some widgets have explicitly set num_audio_formats and some doesn't.

Reply:
some widgets support only 1 format, so they're set in teh class definition. Others support variable number of format based on tplg, so they're set in the top-level file